.github/workflows/build-conda-images.yml

```diff
@@ -32,7 +32,7 @@ concurrency:
 jobs:
   build-docker:
     environment: ${{ (github.ref == 'refs/heads/main' || startsWith(github.event.ref, 'refs/tags/v')) && 'docker-build' || '' }}
-    runs-on: linux.9xlarge.ephemeral
+    runs-on: am2.linux.9xlarge.ephemeral
```

Comment:
is this job actually failing?

Reply:
It was not executed since condition was wrong, but given that manywheel jobs are failing there is a good chance it will fail as well. I will post a followup PR to re-enable these and manywheel jobs to al2023

Reply:
Confirmed these are also failing for conda builds.